src/core_scan/Core_scan.mli

```diff
@@ -86,7 +86,7 @@ val extracted_targets_of_config :
 val select_applicable_rules_for_target :
   analyzer:Xlang.t ->
   products:Semgrep_output_v1_t.product list ->
-  path:Fpath.t ->
+  source:Source.t ->
```

Comment:
nit: I think it's weird that this takes a source rather than a target but I guess it's always been that way

Reply:
I _could_ make it take a whole target, I suppose, but I would rather pass the minimum we really have a foreseeable need for. If it makes you happier I can rename this function.